osquery/main/harnesses/fuzz_utils.cpp

```diff
@@ -17,6 +17,12 @@
 
 namespace osquery {
 
+std::set<std::string> kDisabledFuzzingTables = {
+    "file",
+    "hash",
```

Comment:
hash is interesting... Do we fuzz the hash functions elsewhere?

Reply:
Good question, I was thinking of removing the file enumeration logic too. I'd like to be slightly reactive here and only add alternate paths (fuzz path vs. normal path) selectively.